benchmarks/timeseries/ringbuffer_memusage.py

```diff
@@ -0,0 +1,91 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Memory allocation benchmark for the ringbuffer."""
+
+from __future__ import annotations
+
+import argparse
+import tracemalloc
+from datetime import datetime, timedelta
+
+import numpy as np
+
+from frequenz.sdk.timeseries import Sample
+from frequenz.sdk.timeseries._ringbuffer import OrderedRingBuffer
+
+FIVE_MINUTES = timedelta(minutes=5)
```

Comment:
you might want to define this constant within main() as it seems it is only used there rather than defining it globally

Reply:
Having a constant in a function causes complains with the CI.

Reply:
I see, they work within classes. Resolving...